js/index.js

```diff
@@ -37,6 +37,104 @@ const siteContent = {
   },
 };
 
-// Example: Update the img src for the logo
+// Example: Update the img src for the logo 
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+let nav = document.querySelector('nav');
+
+let navA = document.querySelectorAll('a');
+navA[0].textContent= siteContent["nav"]["nav-item-1"];
+navA[1].textContent= siteContent["nav"]["nav-item-2"];
+navA[2].textContent= siteContent["nav"]["nav-item-3"];
+navA[3].textContent= siteContent["nav"]["nav-item-4"];
+navA[4].textContent= siteContent["nav"]["nav-item-5"];
+navA[5].textContent= siteContent["nav"]["nav-item-6"];
+
+navA[0].style.color= 'green';
+navA[1].style.color= 'green';
+navA[2].style.color= 'green';
+navA[3].style.color= 'green';
+navA[4].style.color= 'green';
+navA[5].style.color= 'green';
+
+var home = document.createElement("a");
+home.innerText = "Home"
+home.href = '#';
+home.style.color='green';
+var blog = document.createElement("a");
+blog.innerText = "Blog";
+blog.href='#';
+blog.style.color = 'green';
+
+nav.prepend(home);
+nav.appendChild(blog);
+
+let h1Selector = document.querySelector('.cta h1');
+h1Selector.textContent = siteContent ["cta"]["h1"];
+
+let buttonSelector = document.querySelector('.cta button');
+buttonSelector.textContent = siteContent["cta"]["button"];
+
+const img1 = document.getElementById('cta-img');
+img1.setAttribute('src', siteContent["cta"]["img-src"]);
+
+//Main
```

Comment:
Nice commenting to let future developers understand the organization of your code.